<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorWindow</class>
 <widget class="QMainWindow" name="MapEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="GLMapWidget" name="openGLWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionDump">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dump</string>
   </property>
  </action>
  <action name="actionExport_sprites_as_PNG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sprites as PNG</string>
   </property>
  </action>
  <action name="actionExport_sprites_as_TGA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sprites as TGA</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDisplay_external_references">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display external references</string>
   </property>
  </action>
  <action name="actionDisplay_shadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display shadow</string>
   </property>
  </action>
  <action name="actionDisplay_ground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display ground</string>
   </property>
  </action>
  <action name="actionPick_background_color">
   <property name="text">
    <string>Pick background color</string>
   </property>
  </action>
  <action name="action_Pick_color">
   <property name="text">
    <string>Pick background color</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timeline</string>
   </property>
  </action>
  <action name="actionImport_SH">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import .SH</string>
   </property>
  </action>
  <action name="actionExport_SH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export .SH</string>
   </property>
  </action>
  <action name="actionExport_sprites_IDs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sprites IDs</string>
   </property>
  </action>
  <action name="actionExport_color_table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export color table</string>
   </property>
  </action>
  <action name="actionSprite_sheet_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sprite sheet view</string>
   </property>
  </action>
  <action name="actionImport_color_table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import color table</string>
   </property>
  </action>
  <action name="actionColor_table_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color table view</string>
   </property>
  </action>
  <action name="actionReset_camera">
   <property name="text">
    <string>Reset camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLMapWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>GLMapWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
